cosmovisor/cmd/cosmovisor/cmd/version.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+// Cosmovisor version value. Set during build
+var Version string
+
+const (
+	versionRegex = `^v(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)`
+	// ^                       - Ensuring the text starts with the match
+	// v                       - Version string starts with `v`
+	// (?P<major>0|[1-9]\d*)   - Match group `major` - Matches 0 once or digits 1-9 zero or many times
+	// \.                      - Dot between `major` and `minor`
+	// (?P<minor>0|[1-9]\d*)   - Match group `minor`- Matches 0 once or digits 1-9 zero or many times
+	// \.                      - Dot between `minor` and `patch`
+	// (?P<patch>0|[1-9]\d*)   - Match group `patch`- Matches 0 once or digits 1-9 zero or many times
+)
+
+func isVersionCommand(args []string) bool {
+	return len(args) == 1 && strings.EqualFold(args[0], "version")
+}
+
+func printVersion() {
+
+	versionFound, version := getVersion(Version)
```

Comment:
Why do we need this check? `Version` works.

Reply:
`Verion` would return something like `v0.1.0-85-g65baacac0`
That would be okay? No need to format the version into `v0.1.0`?

Reply:
I think is ok to get the full version.